staging/src/k8s.io/client-go/util/workqueue/default_rate_limiters.go

```diff
@@ -62,6 +62,54 @@ func (r *BucketRateLimiter) NumRequeues(item interface{}) int {
 func (r *BucketRateLimiter) Forget(item interface{}) {
 }
 
+// ItemBucketRateLimiter implements a workqueue ratelimiter API using standard rate.Limiter.
+// Each key is using a separate limiter.
```

Comment:
I think it'd be good to explain under what conditions it makes sense to use this.